.changeset/spicy-tips-dream.md

```diff
@@ -0,0 +1,10 @@
+---
+'@astrojs/deno': major
+'@astrojs/netlify': major
+'@astrojs/image': major
+'astro': major
+---
+
+**Breaking Change**: client assets are built to an `_astro` directory rather than the previous `assets` directory. This setting can now be controlled by the new `build` configuration option named `assets`.
```

Comment:
Suggestion to add more context re: stuff being all over the place previously, plus some other language tweaks:

```suggestion
**Breaking Change**: client assets are built to an `_astro` directory in the build output directory. Previously these were built to various locations, including `assets/`, `chunks/` and the root of build output.

You can control this location with the new `build` configuration option named `assets`.
```

Sorry to come in after the merge!

Reply:
This is great, will update!